server/historian/packages/historian-base/src/services/definitions.ts

```diff
@@ -57,3 +59,26 @@ export interface ITenant {
 
     storage: IStorage;
 }
+
+/**
+ * An extension of ITenantCustomData.
+ * It is important to include all fields when updating tenant custom data.
+ */
+export interface ITenantCustomDataExternal extends ITenantCustomData {
```

Comment:
These interfaces should probably live in @fluidframework/server-services-core.

Reply:
Ditto. But feel free to do it on a separate PR